src/components/Layout/SideNav.tsx

```diff
@@ -180,18 +181,17 @@ const SelectNetworkButton: React.FunctionComponent<SelectNetworkButtonPropType>
   marginTop,
 }: SelectNetworkButtonPropType) => {
   const theme = useTheme()
-  const activeNetworks = useActiveNetworks()
-  const { isAllChain } = useActiveNetworkUtils()
+  const { isAllChain, networkInfo: activeNetworkInfo } = useActiveNetworkUtils()
```

Comment:
Why do we have to rename `networkInfo` to `activeNetworkInfo`

Reply:
because I only update one place, but ok I will update
<img width="781" alt="image" src="https://user-images.githubusercontent.com/33005392/181306005-94dd27e2-df8a-495c-8c40-91a1ebd5b28a.png">